docs/quick_start.rst

```diff
@@ -0,0 +1,362 @@
+.. role:: bash(code)
+          :language: bash
+
+====================
+Quick Start
+====================
+
+--------------------
+What can LBANN do?
+--------------------
+
+The Livermore Big Artificial Neural Network toolkit (LBANN) is an HPC-centric
+deep learning training framework that works across multiple levels of
+parallelism.  LBANN is capable of taking advantage of HPC hardware to
+accelerate the training of deep learning models on massive datasets.
+
+
+--------------------
+Installing LBANN
+--------------------
+
+LBANN supports installation through Spack and CMake.  We recommend using the
+Spack installation instructions below.  If the Spack install fails, try using
+the :ref:`CMake install <build-with-cmake>`.
+
+1.  Download and install `Spack <https://github.com/llnl/spack>`_.  Enable the
+    additional Spack commands for module files described `here
+    <https://spack.readthedocs.io/en/latest/module_file_support.html#id2>`_:
+
+    .. code-block:: bash
+
+        source ${SPACK_ROOT}/share/spack/setup-env.sh
+
+2.  Users that are `familiar with Spack
+    <https://spack-tutorial.readthedocs.io/en/latest/tutorial_basics.html>`_
+    and already have a `custom Spack ecosystem
+    <https://spack.readthedocs.io/en/latest/configuration.html>`_ can install
+    LBANN with:
+
+    .. code-block:: bash
+
+        spack install lbann <customization options>
+
+    For users new to Spack, LBANN provides a script that will perform some
+    basic configuration (e.g., add paths to externally installed packages) and
+    install LBANN in a Spack environment.  *This script is only tested and
+    maintained for systems at LLNL, NERSC, and ORNL.  If you are not running on
+    a system at one of these institutions, you may try the Spack install above
+    or the :ref:`CMake install <build-with-cmake>`.* To use this installation
+    script, clone the repository and run the script:
+
+    .. code-block:: bash
+
+        git clone https://github.com/llnl/lbann
+        cd ./lbann
+        ./scripts/install_lbann.sh -e lbann
+
+    View other options available by passing the :code:`-h` option to the
```

Comment:
Should the -d option be used here as well? And the -e option should probably be explained so that it's more explicit, and/or the help menu in the script should explain that -e can be used to customize the name of the environment.

Reply:
Actually, it is correct as written here.  If you don't provide the `-d` flag it will build lbann using that environment.  This is what users would want. The `-d` flag is really for developers.